src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstier/service/CommunityApiService.kt

```diff
@@ -2,14 +2,17 @@ package uk.gov.justice.digital.hmpps.hmppstier.service
 
 import org.springframework.stereotype.Service
 import uk.gov.justice.digital.hmpps.hmppstier.client.CommunityApiClient
+import uk.gov.justice.digital.hmpps.hmppstier.domain.Conviction
 import uk.gov.justice.digital.hmpps.hmppstier.domain.DeliusAssessments
 
 @Service
 class CommunityApiService(
   private val communityApiClient: CommunityApiClient,
 ) {
 
-  public fun getDeliusAssessments(crn: String): DeliusAssessments {
-    return DeliusAssessments from communityApiClient.getDeliusAssessments(crn)
-  }
+  fun getDeliusAssessments(crn: String): DeliusAssessments =
+    DeliusAssessments.from(communityApiClient.getDeliusAssessments(crn))
+
+  fun getConvictionsWithSentences(crn: String): List<Conviction> =
+    Conviction.from(communityApiClient.getConvictions(crn).filterNot { it.sentence == null })
```

Comment:
This is kind of conflated with the Conviction constructor which I'm not too keen on